### ISSUE TITLE
--start command not found

### ISSUE THREAD

**Issue by menelic** [potatoeggy/mandown#67]
First of all: Thanks for a great tool

**What broke?**

- [x] Downloader
- [ ] Processor
- [ ] Converter
- [ ] Other

**Describe the bug**
I cant pass the --start and --end commands as described in the docs, it says
[1] 15101
--start: command not found

the command is also not listed when I open the mandown --help manpage

**Steps to reproduce**
If the downloader broke, please include a URL that breaks it.
https://www.webtoons.com/en/challenge/red-origins/list?title_no=57202&page=1

**Additional context**
The series I am downloading is LONG and there are errors after a few chapters. But because I cant limit the download, I also cannot convert the downloaded chapters that work to epub.

**Comment by potatoeggy** [comment]
Thanks for opening an issue!

**TL;DR: spam `mandown get https://www.webtoons.com/en/challenge/red-origins/list?title_no=57202&page=1 --threads=2`**

What's the exact command you're running? `--start` is used with `mandown get`, so you would have something like:

```
mandown get https://www.webtoons.com/en/challenge/red-origins/list?title_no=57202&page=1 --start 1 --end 10
```

I can reproduce your issue with the URL that you provided and am looking into it. In the meantime, you can use the following flag to slow down mandown but make it more reliable (`--threads=2`). Additionally, because Mandown doesn't delete the images it's already downloaded, if you rerun the command after each crash, it'll pick up right where it started until all of the images are downloaded.

**Comment by potatoeggy** [comment]
No longer reproducible in master — if it comes up again, please reopen this issue.
